src/pagination/pagination.spec.ts

```diff
@@ -196,7 +320,7 @@ describe('ngb-pagination', () => {
       fixture.detectChanges();
       expectPages(fixture.nativeElement, ['«', '1', '+2', '3', '»']);
 
-      getLink(fixture.nativeElement, 0).click();
+      getLink(fixture.nativeElement, 1).click();
```

Comment:
Hi here I fix the test that was not testing what we wanted (click on the page number) as previously it was clicking on the previous link (another test is doing it).
BR